ui/components/empty-data.js

```diff
@@ -0,0 +1,3 @@
+export default function EmptyData({ text }) {
```

Comment:
This one would be better staying decoupled vs as a component due to padding/margin

Reply:
hmm but this is the empty data message for the sidebar. So isn't it safe to assume that the padding and margin would be the same for all the pages' sidebar